### DISCUSSION TITLE
Plotting mode shapes 6 DoF problem

### DISCUSSION THREAD

**Discussion by hserdogan94** [petrobras/ross#1089] [Q&A] [answered]
I have an issue when I plot the modes in 2d and 3d. when I plot them in the rotor_example( ), it works and makes sense, but when I plot the mode shapes for 6dof it doesn't make sense, particularly high dof numbers. 
![image](https://github.com/user-attachments/assets/5b544ec8-04bd-45dd-a15c-b9c3196e0e23)

![image](https://github.com/user-attachments/assets/0229312a-b1d4-401f-8179-2cd125f213bc)
This is with high DoF :
![image](https://github.com/user-attachments/assets/22425f2f-ac53-4497-9d50-d5d56567b871)

**Comment by jguarato** [comment, marked as answer]
Hi @hserdogan94!

Thanks for reporting this issue. We are aware of it, however, it has not been addressed yet. We will investigate and hope to solve this soon in a fixed version of ROSS.

For future errors/bugs and issues that you encounter in ROSS, we recommend reporting them as a new [Issue](https://github.com/petrobras/ross/issues),  instead of addressing them here in the discussion area.

Thanks again.

**Reply by hserdogan94** [reply]
Ok thank you so much !
